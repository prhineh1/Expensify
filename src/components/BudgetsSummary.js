import React from 'react';
import { Link } from 'react-router-dom';

const BudgetsSummary = () => (
  <div className='page-header'>
      <div className='content-container'>
          <h1 className='page-header__title'>Budgets</h1>
          <p className='page-header__subtitle'>Note: Budgeted expenses are tracked monthly.</p>
          <div className='page-header__actions'>
              <Link className='button' to='/create/budget'>Add Budget</Link>
              <Link className='button' to='/dashboard/expenses'>View Expenses</Link>
          </div>
      </div>
  </div>
);

export default BudgetsSummary;