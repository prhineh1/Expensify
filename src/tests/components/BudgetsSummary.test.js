import React from 'React';
import { shallow } from 'enzyme';
import BudgetsSummary from '../../components/BudgetsSummary';

test('should render BudgetsSummary', () => {
  const wrapper = shallow(<BudgetsSummary />);
  expect(wrapper).toMatchSnapshot();
});