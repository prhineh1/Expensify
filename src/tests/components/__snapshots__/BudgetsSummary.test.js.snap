// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BudgetsSummary 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Budgets
    </h1>
    <p
      className="page-header__subtitle"
    >
      Note: Budgeted expenses are tracked monthly.
    </p>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        replace={false}
        to="/create/budget"
      >
        Add Budget
      </Link>
      <Link
        className="button"
        replace={false}
        to="/dashboard/expenses"
      >
        View Expenses
      </Link>
    </div>
  </div>
</div>
`;
